ui/StatusQ/src/StatusQ/Components/StatusListItem.qml

```diff
@@ -146,6 +148,9 @@ Rectangle {
             badge.border.color: statusListItem.color
             ringSettings: statusListItem.ringSettings
             loading: statusListItem.loading
+            hoverEnabled: statusListItem.iconHoverEnabled
```

Comment:
the current approach is to use `root` as id for the top-most item. Usually touching that is a good moment to refactor :)